documentation/docs/src/user-guide/ibc.md

```diff
@@ -0,0 +1,137 @@
+
+### Downloading the IBC-rs repo
+
+In order to create the relayer that is able to set up IBC connections, the following repo will need to be cloned. Then we enter the directory.
+
+```bash=
+git clone git@github.com:heliaxdev/ibc-rs.git && cd ibc-rs
+```
+
+
+
+
+### Running multiple instances of Namada
+
+In order to make IBC transfers across two Namada chains, one must be running two nodes of Namada at once. In order to do this, two CHAIN-IDs will need to be retrieved. In the case that there is only 1 testnet running, one may need to setup a local namada chain (which will be proof of authority since you will be the only one running it :P).
+
+Below, I discuss how to setup connections between two local namada testnets, and I append how to achieve ibc package transfers between arbitrary chain_ids afterwards.
+
+#### Setting up local Namada fractal instances
+In order to setup a local testnet, we have created an easy script that does this for you. The following command will setup two local testnets.
+
+In order for this to be completed, one must have cloned the namada directory and built the wasm scripts. This is achieved by the following command
+
+```bash=
+git clone 
+#Make sure you are in the home directory
+cd ~
+#Now clone
+git@github.com:anoma/namada.git && cd namada
+make build-wasm-scripts
+export NAMADA_DIR=${HOME}/namada
+```
+
+The second command requires being located in the achieved in the ibc-rs directory.
+
+
+
+```bash=
+cd ~/ibc-rs/
+./e2e/namada-test.sh ${NAMADA_DIR}
+#Followed by
+cargo run --bin hermes -- -c namada_e2e_config.toml start
+```
+
+Congratulaitons, you now (hopefully) have 2 local namada chains running in the background.
+
+### Opening up communication
+You should now have what is required in order to setup communication channels
+
+We will begin by collecting the CHAIN_IDs for the different blockchains we want to establish a connection between.
+
+A simple way to find these chain ids is to run
+
+```bash=
+ls ~/ibc-rs/.anoma
+```
+
+This should show something like
+
+```bash=
+ls .anoma/
+global-config.toml                   namada-test.d9efda6aeb93cf8c38
+namada-test.79d9d85f3dbfa1afa2       namada-test.d9efda6aeb93cf8c38.toml
+namada-test.79d9d85f3dbfa1afa2.toml
+
+```
+
+In this case, the 2 chain ids of interest are 
+
+```bash!
+export CHAIN_A_ID=namada-test.d9efda6aeb93cf8c38
+export CHAIN_B_ID= namada-test.79d9d85f3dbfa1afa2
+```
+
+:::info
+But the above CHAIN_IDs will depend on your own setup, so do check this for yourself!
+:::
+
+Finally,
+
+```bash!
+export HERMES_CONFIG=
+cargo run --bin hermes -- -c ${HERMES_CONFIG} \
+create channel ${CHAIN_A_ID} --chain-b ${CHAIN_B_ID} \
+--port-a transfer --port-b transfer \
+--new-client-connection
+```
+
+
+This will establish a connection that will allow you to make transfers across chains.
+
+Such transfers can be achieved by
```

Comment:
Can we go ahead and give an example of how to generate test keys for all of these parameters?

Reply:
I'm a bit confused by what you mean by this, generate test keys for transfers?

Reply:
yeah